specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/ManagedCertificate_CreateOrUpdate.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "examplerg",
+    "environmentName": "testcontainerenv",
+    "managedCertificateName": "certificate-firendly-name",
+    "api-version": "2022-06-01-preview",
+    "managedCertificateEnvelope": {
+      "location": "East US",
+      "properties": {
+        "subjectName": "my-subject-name.company.country.net",
+        "domainControlValidation": "CNAME"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/testcontainerenv/managedCertificates/certificate-firendly-name",
+        "type": "Microsoft.App/ManagedEnvironments/managedCertificates",
+        "location": "East US",
+        "properties": {
+          "subjectName": "CN=my-subject-name.company.country.net",
+          "provisioningState": "Succeeded",
+          "domainControlValidation": "CNAME"
+        }
+      }
+    },
+    "201": {
+      "headers": {
+        "azure-asyncoperation": "https://management.azure.com/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/examplerg/providers/Microsoft.App/locations/eastus/managedCertificateOperationStatuses/dc27c9cd-370f-4623-95d4-827b6dc4120a?api-version=2022-06-01-preview&azureAsyncOperation=true"
+      },
+      "body": {
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/testcontainerenv/managedCertificates/certificate-firendly-name",
+        "type": "Microsoft.App/ManagedEnvironments/managedCertificates",
+        "location": "East US",
+        "properties": {
+          "subjectName": "CN=my-subject-name.company.country.net",
+          "provisioningState": "Pending",
+          "domainControlValidation": "CNAME"
+        }
+      }
+    },
+    "400": {
+      "code": "ManagedCertNoUpdate",
+      "message": "Properties of managed certificates cannot be updated. Please delete the current resource and retry or use a different resource id."
+    }
```

Comment:
what happens on a Re-PUT ? REST operations need to be idempotent you cannot fail a RE-PUT